notebooks/features/vw/Vowpal Wabbit - Contextual Bandits.ipynb

```diff
@@ -0,0 +1,204 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img width=\"200\" src=\"https://mmlspark.blob.core.windows.net/graphics/emails/vw-blue-dark-orange.svg\" />\n",
+    "\n",
+    "# Contextual-Bandits using Vowpal Wabbit\n",
+    "\n",
```

Comment:
nit: COnsider explaining or linkign to  explanation of what contexctual bandits are and when to use them